core/settings/base.py

```diff
@@ -34,6 +34,10 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'apps.api',
+    'apps.account',
+    'apps.shared',
+    'apps.rating',
+    'apps.timetable',
 ]
```

Comment:
@andela-snwuguru I forgot to add this earlier, about separating apps settings. We currently have all the apps in `installed_apps`, but it would be nicer if we separate them this way for example.....
```
PREREQ_APPS = [
   ‘django.contrib.auth’,
   ‘django.contrib.contenttypes’,
   …
]

PROJECT_APPS = [
  'apps.api',
   'apps.accounts',
  'apps.shared',
  'apps.ratings',
  'apps.timetables'
]

INSTALLED_APPS = PREREQ_APPS + PROJECT_APPS
```